scripts/globals/conquest.lua

```diff
@@ -1471,8 +1471,6 @@ xi.conquest.onConquestUpdate = function(zone, updatetype)
             end
         end
     end
-
-    xi.garrison.buildNpcTable(zone)
```

Comment:
Looks like this accidentally reverted a garrison change.

Reply:
After speaking with Zach, this was intentional.  That buildNpcTable() function was not implemented in base